docker/usr/local/package/opencl-clang.sh

```diff
@@ -40,14 +40,14 @@ mkdir -p build
 cd build
 
 # Configure using cmake
-cmake .. \
+cmake .. "$@" \
```

Comment:
I would leave the standard arguments out in the docker file and continue to pass them in like this.
Hiding things in the script files makes it harder to read